json_rpc/clients/httpclient.nim

```diff
@@ -16,7 +16,7 @@ type
 
 const
   MaxHttpHeadersSize = 8192        # maximum size of HTTP headers in octets
-  MaxHttpRequestSize = 128 * 1024  # maximum size of HTTP body in octets
+  MaxHttpResponseSize = 128 * 1024  # maximum size of HTTP body in octets
```

Comment:
where is this limit coming from? it looks rather arbitrary - ie requesting a block or state over RPC will easily exceed this limit

Reply:
This PR was a direct response to https://github.com/status-im/nimbus-eth2/issues/1650 
Perhaps it should be way bigger by default and should also be configurable (as opposed to being a simple const)

Reply:
Indeed - it is not a practical solution to use a constant here - the constant could be a default, but there must be an override at least per url or it won't work for several actual requests we issue (and be unnecessarily lax for others)